SparkPost/2web2ui
# FE-1021: Filter internal test users from Pendo

### What Changed
 - Set isInternalTestUser metadata for Pendo

### How To Test
 - From Chrome devtool's console, run `pendo.validateInstall()`
 - When logged in as "x@sparkpost.com" user, should be true
 - When logged in as "x@example.com" user, should be false
